src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -13,6 +13,13 @@ public partial class MauiAppCompatActivity : AppCompatActivity
 
 		protected override void OnCreate(Bundle? savedInstanceState)
 		{
+			// https://github.com/dotnet/maui/issues/24742
+			// By default we opt out of edge to edge enforcment on Android 35+
+			// Must be applied before the DecorView is setup
+			// We set force to false in case the style has already been explicitly
+			// applied to disable opting out of edge to edge enforcement
+			Theme?.ApplyStyle(Resource.Style.OptOutEdgeToEdgeEnforcement, force: false);
```

Comment:
Would it work to set this in `Maui.MainTheme`:
```xml
<item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>
```
This would avoid a C# -> Java call on startup.